expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.coroutines.*
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.collectLatest
 
-class GoogleMapsView(context: Context): LinearLayout(context), OnMapReadyCallback {
+class GoogleMapsView(context: Context, private val gestures: MapsViewGestures?): LinearLayout(context), OnMapReadyCallback {
 
   private val mapView: MapView = MapView(context)
   private lateinit var googleMap: GoogleMap
```

Comment:
In my opinion there is no need to pass gestures object through constructor. We can just make it class field as we did with other values. I would also use `lateinit` variable to stick to the convention we used before. Additionally I think we can type gestures used here in `GoogleMapsView` as `GoogleMapsGestures`, because this is specific implementation of gestures we have for Google Maps. What are your thoughts?
```suggestion
class GoogleMapsView(context: Context): LinearLayout(context), OnMapReadyCallback {

  private val mapView: MapView = MapView(context)
  private lateinit var googleMap: GoogleMap
  private lateinit var gestures: GoogleMapsGestures
```

Reply:
All the other things you commented on are done the way they are because I wanted this injection through constructor. But you may be right, there is not much of implementation freedom to classes like GoogleMapsGestures, that would make anyone want to replace it with something different. In return we gain more clarity about the way things are instantiated.